src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -0,0 +1,175 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/blocks';
+import SimpleSelect from '.';
+import Option from '../option';
+import SelectTextbox from '../select-textbox'
+
+<Meta title="Design System/Select" />
+
+# Simple Select
+
+Select one of available options from the drop-down menu
+
+## Overview 
+
+Simple Select is a Carbon styled equivalent of HTML Select Element.
+
+## Guidance, hints and tips 
+
+Always insert `Option` Components inside the `SimpleSelect`, analogous to the original `<select>` and `<option>` HTML Elements.
+
+## Quick Start 
+
+To use the `SimpleSelect` component you have to import two components: `<SimpleSelect />` and `<Option />`
```

Comment:
We use the term `SimpleSelect` in this document but we changed the public interface to be `Select`. We need to update this document to reflect that change.

e.g. `import {SimpleSelect} from 'carbon-react/lib/components/select'` is actually `import {Select} from 'carbon-react/lib/components/select'`